tests/infra/crypto.py

```diff
@@ -79,6 +79,25 @@ def generate_rsa_keypair(key_size: int) -> Tuple[str, str]:
     return priv_pem, pub_pem
 
 
+def generate_ec_keypair(curve_name: str) -> Tuple[str, str]:
+    if curve_name == "secp256k1":
```

Comment:
I believe we've dropped support for secp256k1?

Reply:
That's right, we removed that. Currently we support only secp384r1 and secp256r1. In general JS applications might want to use it for other purposes in their application, so it would still make a little bit of sense to keep in the API, but it's not necessary.

Reply:
I switched to `secp256r1`.